src/agent/pythd/api.rs

```diff
@@ -897,14 +951,14 @@ pub mod rpc {
             let received_json = test_client.recv_json().await;
 
             // Check that the result is what we expect
-            let expected_json = r#"{"jsonrpc":"2.0","error":{"code":-32603,"message":"Missing request parameters","data":null},"id":5}"#;
+            let expected_json = r#"{"jsonrpc":"2.0","error":{"code":-32603,"message":"Error handling request. Check pyth-agent logs for details.","data":null},"id":5}"#;
```

Comment:
I don't like this new error response behavior. The old messages were nicer for the user, because they could debug their requests more easily. I suggest reverting that change.

Reply:
Reverted back to full error messages